--- Descriptive.pm~	Tue Apr 27 00:25:26 1999
+++ Descriptive.pm	Sat Nov  6 17:16:42 1999
@@ -2,7 +2,7 @@
 
 ##This module draws heavily from perltoot v0.4 from Tom Christiansen.
 
-require 5.00404;  ##Yes, this is underhanded, but makes support for me easier
+require 5.004_04;  ##Yes, this is underhanded, but makes support for me easier
 		  ##Not only that, but it's the latest "safe" version of
 		  ##Perl5.  01-03 weren't bug free.
 $VERSION = '2.4';
